Question-1/TableViewDemo/ItemStore.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ItemStore.swift
+//  TableViewDemo
+//
+//  Created by Bharath Reddy on 25/11/22.
+//
+
+import UIKit
+//import Item
+class ItemStore: NSObject {
```

Comment:
Use more specific class name in-order to get the idea of what is inside the class just by reading its name.